src/components/custom/QrReader.vue

```diff
@@ -133,12 +133,29 @@
 	</div>
 </template>
 <script>
-import QrScanner from 'qr-scanner'
+// The BarcodeDetector Web API is not yet supported in all browsers,
+// and "qr-scanner" defaults to a suboptimal implementation if it is not available.
+// The following import makes a better implementation available that is based on a
+// WebAssembly port of ZXing:
+import { setZXingModuleOverrides } from 'barcode-detector'
 import { wait } from '../../lib/utils.js'
 import logger from '../../lib/logger'
 
 const log = logger.get('QrReader')
 
+import QrScanner from 'qr-scanner'
+
```

Comment:
Double check why this import is now here. It might not be needed.

Reply:
It is actually needed: https://github.com/zwave-js/zwave-js-ui/blob/f909f84185cdad7bcf6e8b7d37c65da1b781cad2/src/components/custom/QrReader.vue#L290

Reply:
Yeah but why was it not there before the PR?

Reply:
it was on the first line, check the diff :) 

![2024-11-04_10-59](https://github.com/user-attachments/assets/9d12bb62-c563-4f22-aa5d-68b4b8eb8c64)

Reply:
Ohh, yeah. I was blind.